src/app-directory/README.md

```diff
@@ -1,7 +1,7 @@
 # Application Directory API -- THIS IS NOT A RATIFIED FDC3 API or SPECIFICATION
 
 **This is NOT a ratified API or specification but rather a tool to evaluate both the use cases and technology implementations.**
-[View FULL SPECIFICATION HERE](https://fdc3-app-directory.finos.org/)
+[View FULL FDC3 SPECIFICATION HERE](https://fdc3.finos.org/)
 
 ## Overview
```

Comment:
This overview is also slightly out of context now, should this specific README file be more about the details of the app-directory src files and code generation now @ftbb?